test/e2e/images_test.go

```diff
@@ -45,18 +45,6 @@ var _ = Describe("Podman images", func() {
 		Expect(session.LineInOuputStartsWith("quay.io/libpod/busybox")).To(BeTrue())
 	})
 
-	It("podman images with no images prints header", func() {
-		rmi := podmanTest.PodmanNoCache([]string{"rmi", "-a"})
```

Comment:
This means we can no longer test this functionality?  This seems bad?

Reply:
it got moved to a system test and is in Ed's pr